src/components/search/results/TabViews.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { BsListUl, BsMap, BsCalendar3Week } from 'react-icons/bs';
+import EventList from './EventList';
+import styles from '../../../styles/resultList.module.css';
+
+const TabViews = () => {
+  const [view, setView] = useState(1);
+  const changeView = (index) => {
+    setView(index);
+  };
```

Comment:
By the way, this is the same as simply writing `const changeView = setView`. Think about it from a "leading-the-interpreter" mindset. Which means you don't even need `changeView` at all, just `setView`.